examples/with-typescript-graphql/.graphql-let.yml

```diff
@@ -1,7 +1,8 @@
-schema: lib/type-defs.graphqls
+schema: '**/*.graphqls'
+schemaEntrypoint: 'lib/type-defs.graphqls'
 documents: '**/*.graphql'
 plugins:
   - typescript
   - typescript-operations
   - typescript-react-apollo
-respectGitIgnore: true
+cacheDir: __generated__
```

Comment:
Some `.tsx` will go in this dir. The default `cacheDir` is `node_modules/graphql-let/__generated__`, but Babel doesn't work on files in `node_modules` so I need get them out.